src/entt/meta/node.hpp

```diff
@@ -198,8 +198,8 @@ template<typename... Args>
     return value(context);
 }
 
-[[nodiscard]] inline const void *try_cast(const meta_context &context, const meta_type_node &from, const meta_type_node &to, const void *instance) noexcept {
-    if((from.info != nullptr) && (to.info != nullptr) && *from.info == *to.info) {
+[[nodiscard]] inline const void *try_cast(const meta_context &context, const meta_type_node &from, const type_info *to, const void *instance) noexcept {
+    if((from.info != nullptr) && (to != nullptr) && *from.info == *to) {
```

Comment:
I wonder if we can make it a `const type_info &` instead.
`can_cast` is the only one that could pass a `nullptr` here, but we can prevent it evantually.
`try_cast` would benefit from less checking instead.

Reply:
Good idea 👍